console/network/src/canary_v0.rs

```diff
@@ -136,17 +136,23 @@ impl Network for CanaryV0 {
     /// A list of (consensus_version, block_height) pairs indicating when each consensus version takes effect.
     /// Documentation for what is changed at each version can be found in `N::CONSENSUS_VERSION`
```

Comment:
This commend declaring where the consensus version documentation lives should be updated